app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -10,3 +10,4 @@
 <%= presenter.attribute_to_html(:related_url, render_as: :external_link) %>
 <%= presenter.attribute_to_html(:resource_type, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:source) %>
+<%= presenter.attribute_to_html(:rights_statement) %>
```

Comment:
This renderer uses the [license service](https://github.com/projecthydra-labs/hyrax/blob/master/app/services/hyrax/license_service.rb), and we can hook a new renderer up to the existing [rights statement service](https://github.com/projecthydra-labs/hyrax/blob/master/app/services/hyrax/rights_statements.rb).